packages/icons/src/components/CategoryIcon/Icon.tsx

```diff
@@ -35,14 +19,18 @@ export const Icon = ({
   variant = 'primary',
   disabled,
   children,
-}: IconProps) => (
-  <StyledIcon
-    disabled={disabled}
-    height="20"
-    variant={variant}
-    viewBox="0 0 20 20"
-    width="20"
-  >
-    {children}
-  </StyledIcon>
-)
+  className,
+}: IconProps) => {
+  const computedVariant = `${variant}${disabled ? 'Disabled' : ''}`
+
+  return (
+    <svg
+      className={`${className ? `${className} ` : ''}${categoryIcon[computedVariant as keyof typeof categoryIcon]}`}
```

Comment:
with this https://github.com/scaleway/ultraviolet/pull/5481/files#r2310432594,

 you can remove the casting